test/e2e/networking.go

```diff
@@ -18,24 +18,25 @@ package e2e
 
 import (
 	"fmt"
-	"net/http"
-	"strings"
-	"time"
-
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 	"k8s.io/kubernetes/pkg/api"
 	"k8s.io/kubernetes/pkg/fields"
 	"k8s.io/kubernetes/pkg/labels"
 	"k8s.io/kubernetes/pkg/util"
-
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/pkg/util/wait"
+	"net/http"
+	"strconv"
+	"strings"
+	"time"
 )
 
+const nettestVersion = "1.6"
+const pollTimeout = 60
```

Comment:
this isn't a great name. also, it's unclear where the 60 came from, or what unit it's even supposed to be.